en/core-api/overview.md

```diff
@@ -35,7 +36,7 @@ Choose any of one integration method below that is suitable to your liking:
 #### [Cardless Credit Integration Guide](/en/core-api/cardless-credit.md)
 </div>
 
-### B) Follow Sample Code {docsify-ignore}
+### 2. Follow Sample Code {docsify-ignore}
```

Comment:
Ok so you are going to use `1. 2.` instead of `A) & B)` for non-sequential choices? that's fine as long as you keep it consistent across all the docs. Otherwise, you should leave it be.